src/scenes/Login/_login.scss

```diff
@@ -0,0 +1,20 @@
+.logreg-forms {
+    padding-top: 20px;
+    margin: auto;
+    width: 500px;
+    background-color: #ffffff;
+    box-shadow: 0 7px 7px #555, 0 12px 40px #81d6f0;
+    transition: all 0.3s cubic-bezier(.25,.8,.25,1);
+    border-radius: 15px;
+    height: 350px; 
+    .form-signin {
+        margin: auto 30px;
+        h3{
+            text-align: center;
+        }
+    }
+}
+.welcome-background {
+    background-color: #90b5ec;
+    margin-top: 150px;
+}
```

Comment:
Remember to always add the new line at the end of file

Reply:
yes :>
![image](https://user-images.githubusercontent.com/62316127/123147721-fdfc1780-d424-11eb-8524-075d7c5be338.png)